communities/edinburgh-tech-learners/community-assessment.md

```diff
@@ -0,0 +1,120 @@
+
+### Goals
+
+> - Why do people get involved in your community?
+
+People will get involved in this community because they will benefit from the friendly, supportive environment we will aim to provide. They will have the opportunities to make friends, develop their skills, and help others develop theirs.
+
+> - Are they students hoping to get a degree?
+> - Are they trying to ramp-up to get a job?
+> - Are they passionate about developing, regardless of their major?
+
+Participants may be trying to learn skills to help them get their degree or a job, but ultimately I think the people who will participate are those who are passionate about learning about tech.
+
+> - What activities and goals can you set for your community based on their existing goals?
+
+My main idea currently is to provide regular open workshop sessions where people can attend and support each other in their learning. I'm very keen to provide a collaborative environment rather than a student/teacher approach, since that's what I feel is currently lacking here.
+
+### Location
+
+> - Where does your community currently gather?
+> - Is it an online-only group so far, or is there a place where people can meet?
+> - Is there parking, and if so, is it free?
+> - Is there public transportation available to and from meet-ups?
+> - Is the location accessible to those both on and off campus?
+> - How much does it cost to use the space?
+> - How flexible is the space for different types of presentations and meetings?
+> - How many people can it fit?
+> - Is there comfortable seating for all shapes and sizes?
+> - Is there water or snacks available nearby?
+
+I think the community doesn't currently gather extensively, with the possible exception of people who attend the monthly student tech meetups provided by CompSoc.
+
+In the ideal version of the community, there are regular physical gatherings of people so they have dedicated time to focus on socialising and developing their skills. Ideally, this would be at the central campus of the university — it's a great central location in Edinburgh that should be accessible to students and non-students (the University of Edinburgh is not a campus university — it's very much integrated within the city, all over). Using university venues would provide have many benefits — they already put a lot of effort into making sure people are able to get there and providing a variety of different rooms that could be chosen based on our exact requirements (type of event, number of participants, seating, etc.). If we partnered with a university society, we should be able to use the space for free as well. Water is readily available in most university buildings, and snacks could easily be provided through small donations or sponsorship.
+
+If there was demnad, it might be possible to expand and hold similar events in other places, such as offices of companies that were willing to provide a space in return for some sort of sponsorship.
```

Comment:
Offices is interesting, you have some great companies in Edinburgh, and it adds extra incentive to those involved